neuralmonkey/processors/helpers.py

```diff
@@ -1,10 +1,16 @@
 from typing import Any, Callable, Generator, List
 
+import numpy as np
+
 
 def preprocess_char_based(sentence: List[str]) -> List[str]:
     return list(" ".join(sentence))
 
 
+def preprocess_add_noise(sentence: List[str]) -> List[str]:
+    return np.random.permutation(sentence)
```

Comment:
Tohle jsme nechtěli - chtěli jsme to, co je v branchi mtm18 (random swap dvou sousedních slov)